### PR TITLE
chore: add validation func on organization arg-spec

### PR DESCRIPTION
**Pull request by loicbourgois**

### CODE REVIEW

File: internal/core/validate.go [hunk 1]
```diff
@@ -118,6 +118,16 @@ func ValidateSecretKey() ArgSpecValidateFunc {
 }
 
 func ValidateOrganisationID() ArgSpecValidateFunc {
```
Comment (kindermoumoute) [inline internal/core/validate.go]: ValidateOrganizationID

File: internal/core/validate.go [hunk 2]
```diff
@@ -118,6 +118,16 @@ func ValidateSecretKey() ArgSpecValidateFunc {
...
+	return func(argSpec *ArgSpec, valueI interface{}) error {
+		value := valueI.(string)
+		if value == "" {
+			return nil
+		}
+		return ValidateOrganisationIDRequired()(argSpec, valueI)
+	}
+}
+
+func ValidateOrganisationIDRequired() ArgSpecValidateFunc {
```
Comment (kindermoumoute) [inline internal/core/validate.go]: can you comment: why is this so complicated to validate organization ID? Why do we need 2 functions? Can't we just validate a UUID?

**Comment by jerome-quere** [comment]
Di you check all the places `ValidateOrganisationID` was used before renaming method ?

File: internal/core/validate.go (hunk 2 above)
Comment (kindermoumoute) [inline internal/core/validate.go]: Why is this called required? This arg spec is never required.

**Pull request merged**